src/custom/components/SwapWarnings/index.tsx

```diff
@@ -58,26 +78,26 @@ const WarningContainer = styled(AuxInformationContainer).attrs((props) => ({
     font-size: 14px;
     font-weight: 500;
 
-    svg {
-      stroke: ${({ theme, textColour }) => textColour || theme.infoText};
+    > svg:first-child {
+      stroke: var(--warningColor);
     }
   }
 `
 
 const ErrorStyledInfoIcon = styled(StyledInfoIcon)`
   color: ${({ theme }) => theme.infoText};
 `
-const HIGH_TIER_FEE = { level: '30', colour: '#FF7676' }
```

Comment:
Moving the color selection to the styled component. Instead only feed the `level` number.